### ISSUE TITLE
Expose Integration Flow Endpoint as API and Test the Flow

### ISSUE THREAD

**Issue by sejopa** [SAPDocuments/Issues#608]
Tutorials: https://developers.sap.com/tutorials/cp-starter-isuite-api-management.html
--------------------------

I have not been able to run the "try-out" option from API Portal, it always returns a 401 or 500. I have been able to run the iflow with the api policy credentials.

An even more detailed step would be nice.

Best Regards,
Sergi

**Comment by Karunaharan** [comment]
Hello @sejopa,
Could you please let me know where exactly you face the issue?
In step 2.13, you are expected to see a 401 response. While step 5.2 onwards, you must see a 200 response.

Regards,
Karuna

**Comment by sejopa** [comment]
Hi Karunaharan, 

Nice to meet you and thanks for replying so quickly.

In step 5.3, after filling the policies with cliendid/secret/apitoken I kept getting a 401. Now I just tried again and I get a 500 with the following result:

{
  "fault": {
    "faultstring": "Execution of ServiceCallout getoauthtoken failed. Reason: ResponseCode 401 is treated as error",
    "detail": {
      "errorcode": "steps.servicecallout.ExecutionFailed"
    }
  }
}

Just comment that in step 4.5, instead of seeing the artifact name with "Cloud_Paltform_Connectivity" I actually see "Business Technology Platform Connectivity".
 
I look forward to hearing from you.
Sergi
